icu4c/source/common/uniset_props.cpp

```diff
@@ -105,13 +105,34 @@ static UBool U_CALLCONV uset_cleanup(void) {
     return TRUE;
 }
 
+#ifdef __SUNPRO_CC
+void U_CALLCONV createUni32Set(UErrorCode &errorCode) {
```

Comment:
Since these functions are moving outside of the anonymous C++ namespace, should they be marked as `static`?

Reply:
the one below it is a public/versioned API (expands to `uniset_getUnicode32Instance_63`)